### PR TITLE
Translate aTox to German

### PR DESCRIPTION
**Pull request by robinlinden**
This is a contribution from a user that wished to stay anonymous.

### CODE REVIEW

File: atox/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    <string name="copy_atox">Kopieren</string> <!-- androidx.preference:preference has stolen copy -->
+    <string name="create">Erstellen</string>
+    <string name="create_a_tox_profile">Tox Profil erstellen</string>
+    <string name="delete_contact">Kontakt löschen</string>
+    <string name="message">Nachricht</string>
+    <string name="messages">Nachrichten</string>
+    <string name="messages_incoming">Eingehende Nachrichten</string>
+    <string name="name">Name</string>
+    <string name="name_default">aTox Benutzer</string>
+    <string name="no_contacts_call_to_action">Worauf wartest du? Füge einen Freund hinzu und starte sofort! </string>
```
Comment (avoidr) [inline atox/src/main/res/values-de/strings.xml:20]: Trailing space at the end of the string. Is this intended?

File: atox/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    <string name="update">Aktualisieren</string>
+    <string name="quit">Beenden</string>
+    <string name="quit_confirm">Bist du sicher, dass du dich ausloggen willst?</string>
+    <!-- Friend request notification -->
+    <string name="friend_requests">Freundschaftsanfragen</string>
+    <string name="friend_request_from">Freundschaftsanfrage von %1$s</string>
+    <!-- Chat history clearing -->
+    <string name="clear_history">Verlauf löschen</string>
+    <string name="clear_history_cleared">Verlauf gelöscht</string>
+    <string name="clear_history_confirm">Bist du sicher, dass du den Verlauf mit %1$s löschen möchtest? </string>
```
Comment (avoidr) [inline atox/src/main/res/values-de/strings.xml:44]: Trailing space at the end of the string. Is this intended?

Comment (avoidr) [reply atox/src/main/res/values-de/strings.xml:44]: Also, "möchtest" and "willst" are synonyms. We have the latter twice, but the former is used once. I suggest changing to "willst" for consistency.

File: atox/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    <string name="export_file_failure">Etwas ist beim Export schief gegangen. Pre-0.5.1 Übertragung wurde standardmäßig exportiert.</string>
+    <string name="draft_message">Entwurf: %1$s</string>
+    <string name="pref_heading_address">Adresse</string>
+    <string name="pref_heading_port">Port</string>
+    <string name="pref_heading_proxy">Proxy</string>
+    <string name="warn_proxy_broken">Falscher Proxy konfiguriert. Vor Verwendung der App beheben oder deaktivieren.</string>
+    <string name="proxy_not_found">Proxy nicht gefunden</string>
+    <string name="bad_type">Defekter Typ</string>
+    <string name="bad_host">Defekter Host</string>
+    <string name="bad_port">Defekter Port</string>
```
Comment (avoidr) [inline atox/src/main/res/values-de/strings.xml:114]: "Defective" sounds like something is broken. Probably "invalid" is better, so: "Ungültiger" for all three strings.

File: atox/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    <string name="no_contacts_call_to_action">Worauf wartest du? Füge einen Freund hinzu und starte sofort! </string>
+    <string name="navigation_drawer_close">Navigationsschublade schließen</string>
+    <string name="navigation_drawer_open">Navigationsschublade öffnen</string>
+    <string name="profile_photo_description">Profilbild</string>
+    <string name="reject">Ablehnen</string>
+    <string name="send">Senden</string>
+    <string name="sending">Wird gesendet…</string>
+    <string name="settings">Einstellungen</string>
+    <string name="status_indicator">Statusanzeige</string>
+    <string name="status_message">Status Nachricht</string>
```
Comment (avoidr) [inline atox/src/main/res/values-de/strings.xml:29]: This one is inconsistent with `profile_status_message`. Maybe change this one to "Statusmeldung".

File: atox/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    <string name="status_dialog_edit_user_status">Status ändern</string>
+    <string name="mimetype_handler_not_found">Es konnte keine Anwendung gefunden werden, welche diesen Dateityp öffnen kann %1$s</string>
+    <string name="attach_file">Datei anhängen</string>
+    <string name="delete">Löschen</string>
+    <string name="notification_file_transfer">Eingehende Datenübertragung \"%1$s\"</string>
+    <string name="cancelled">Abgebrochen</string>
+    <string name="completed">Abgeschlossen</string>
+    <string name="save_as">Speichern unter…</string>
+    <string name="export_file_success">Datei erfolgreich gesendet</string>
+    <string name="export_file_failure">Etwas ist beim Export schief gegangen. Pre-0.5.1 Übertragung wurde standardmäßig exportiert.</string>
```
Comment (avoidr) [inline atox/src/main/res/values-de/strings.xml:105]: `s/schief gegangen/schiefgegangen/`
https://www.korrekturen.de/wortliste/schiefgehen.shtml

File: atox/src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+    <string name="read_qr">QR</string>
+    <string name="pref_heading_theme">Design</string>
+    <string-array name="pref_theme_options">
+        <item>Wie System</item>
+        <item>Hell</item>
+        <item>Dunkel</item>
+    </string-array>
+    <string name="pref_heading_type">Typ</string>
+    <string-array name="pref_proxy_type_options">
+        <item>Keinen</item>
```
Comment (avoidr) [inline atox/src/main/res/values-de/strings.xml:64]: I'm on the fence with this one. Somehow "Keinen" looks strange. Maybe "Keins". Or "Kein". I don't know.

File: atox/src/main/res/values-de/strings.xml [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+        <item>HTTP</item>
+        <item>SOCKS5</item>
+    </string-array>
+    <string name="version_display">aTox v%1$s - %2$d</string>
+    <string name="send_as_action">Als Aktion senden</string>
+    <string name="or_import_existing_save">Oder Importiere ein bestehendes Tox Profil</string>
```
Comment (avoidr) [inline atox/src/main/res/values-de/strings.xml:70]: `s/Importiere/importiere/`

File: atox/src/main/res/values-de/strings.xml (hunk 5 above)
Comment (avoidr) [reply atox/src/main/res/values-de/strings.xml:105]: `s/Export/Exportieren/` will be consistent with "Importieren" in `import_tox_save_failed`.

File: atox/src/main/res/values-de/strings.xml [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+    <string name="draft_message">Entwurf: %1$s</string>
+    <string name="pref_heading_address">Adresse</string>
+    <string name="pref_heading_port">Port</string>
+    <string name="pref_heading_proxy">Proxy</string>
+    <string name="warn_proxy_broken">Falscher Proxy konfiguriert. Vor Verwendung der App beheben oder deaktivieren.</string>
+    <string name="proxy_not_found">Proxy nicht gefunden</string>
+    <string name="bad_type">Defekter Typ</string>
+    <string name="bad_host">Defekter Host</string>
+    <string name="bad_port">Defekter Port</string>
+    <string name="tox_save_export_failure">Export fehlgeschlagen: %1$s</string>
```
Comment (avoidr) [inline atox/src/main/res/values-de/strings.xml:115]: String name... inconsistent with
```
import_tox_save
import_tox_save_failed
export_tox_save
```
?

File: atox/src/main/res/values-de/strings.xml [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+    <string name="add_contact">Kontakt hinzufügen</string>
+    <string name="add_contact_message_error_empty">Nachricht darf nicht leer sein</string>
+    <string name="confirm">Bestätigen</string>
+    <string name="cancel">Abbrechen</string>
+    <string name="connecting">Verbindet…</string>
+    <string name="contact_typing">tippt…</string>
+    <string name="copied">Kopiert</string>
+    <string name="copy_atox">Kopieren</string> <!-- androidx.preference:preference has stolen copy -->
+    <string name="create">Erstellen</string>
+    <string name="create_a_tox_profile">Tox Profil erstellen</string>
```
Comment (avoidr) [inline atox/src/main/res/values-de/strings.xml:13]: We have "Tox-ID", which is good. Then we probably should have "Tox-Profil".
So: `s/Tox Profil/Tox-Profil/g`

**Review by avoidr — COMMENTED** [review]
>You need to leave a comment indicating the requested changes.

Well that's new and annoying...

File: atox/src/main/res/values-de/strings.xml (hunk 8 above)
Comment (robinlinden) [reply atox/src/main/res/values-de/strings.xml:115]: Yeah, I've sadly been terrible at keeping these consistent. :( I'll clean all of them up in a separate commit when I feel like it. Opened an issue to track it: https://github.com/evilcorpltd/aTox/issues/629

**Pull request merged**